tests/test_io/test_csv.py

```diff
@@ -0,0 +1,105 @@
+import pandas as pd
+import pytest
+
+from flood_adapt.object_model.io.csv import read_csv
+
+CSV_CONTENT_HEADER = """time,data_0
+2023-01-01,1.0
+2023-01-02,2.0
+2023-01-03,3.0
+"""
+
+CSV_CONTENT_NO_HEADER = """2023-01-01,1.0
+2023-01-02,2.0
+2023-01-03,3.0
+"""
+
+CSV_CONTENT_EMPTY = ""
+
+CSV_CONTENT_INVALID_DATETIME = """time,data_0
+invalid_date,1.0
+2023-01-02,3.0
+"""
+
+CSV_CONTENT_NO_DATA_COLUMNS = """time
+2023-01-01
+2023-01-02
+2023-01-03
```

Comment:
these are only used in theri respective tests so I think it makes more sense to define them there instead of here